Mathlib/Algebra/Order/Interval/Set/Instances.lean

```diff
@@ -123,21 +125,23 @@ theorem mul_le_right {x y : Icc (0 : α) 1} : x * y ≤ y :=
 instance monoidWithZero : MonoidWithZero (Icc (0 : α) 1) := fast_instance%
   Subtype.coe_injective.monoidWithZero _ coe_zero coe_one coe_mul coe_pow
 
-instance commMonoidWithZero {α : Type*} [OrderedCommSemiring α] :
-    CommMonoidWithZero (Icc (0 : α) 1) := fast_instance%
+instance commMonoidWithZero {α : Type*} [CommSemiring α] [PartialOrder α] [IsOrderedRing α] :
+    CommMonoidWithZero (Icc (0 : α) 1) :=
```

Comment:
I don't understand `fast_instance%` so I don't understand why it was removed here (and in 7 or so other places in this file)